data/json/monsters/zed_lieutenant.json

```diff
@@ -0,0 +1,239 @@
+[
+  {
+    "id": "mon_lieutenant_shadow",
+    "type": "MONSTER",
+    "name": { "str_sp": "A shadow?" },
+    "description": "Something is there!  Against all reason, this shadow lurks at the corners of your vision.  You close your eyes, but still, it is there.  No matter how far you run, it is there.  In pitchest black, it is there.  Now you will know why you fear the night.",
```

Comment:
```suggestion
    "description": "Something is there!  Against all reason, this shadow lurks at the corners of your vision.  You close your eyes, but still, it is there.  No matter how far you run, it is there.  In pitchest black, it is there.  Now you know why you fear the night.",
```

Reply:
Politely decline on this one, thank you for the suggestion.